extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -108,6 +128,80 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
     }
     this.subscriptionService_.getDialog().close();
   }
+
+  /**
+   * @return {!Element}
+   * @private
+   * TODO(#37285): add "next story" button to the banner
+   */
+  renderSubscriptionsDialogTemplate_() {
+    return (
+      <div subscriptions-dialog subscriptions-display="NOT granted">
+        <div class="i-amphtml-story-subscriptions-dialog-banner"></div>
+        <div class="i-amphtml-story-subscriptions-dialog-content">
+          <div class="i-amphtml-story-subscriptions-price">
+            {this.element.getAttribute('price')}
+          </div>
+          {!!this.element.getAttribute('title') && (
```

Comment:
Why not use `hasAttribute`? Does thee `!!` cover cases on if it has the attribute and it's nullish? Like an empty string?

Reply:
Yes that's why I am using getAttribute. This way null or empty string would return false. And actually the !! conversion is not needed so I just got it removed.